builder.toml

```diff
@@ -90,5 +94,9 @@ version = "0.4.0"
 
 [[order]]
   [[order.group]]
-    id = "heroku/nodejs"
-    version = "v0.0.5.0.2"
+    id = "heroku/nodejs-engine-buildpack"
+    version = "0.2.0"
+
+  [[order.group]]
+    id = "heroku/nodejs-npm-buildpack"
+    version = "0.0.2"
```

Comment:
i'm surprised that this didn't have/need the heroku/procfile buildpack 🤔

Reply:
It looks like the test is just checking if the image is successfully build, so I should make sure that it works with the Procfile buildpack too (and add it). I've just been using docker-cli locally for runtime commands

Reply:
ok, i don't think it's absolutely necessary as part of this PR since it wasn't there before.